website/_redirects

```diff
@@ -65,11 +65,11 @@
 
 # Old old Gitbook links?
 /docs/introduction/CoreConcepts.html        /introduction/core-concepts
```

Comment:
:exclamation: These are the very old legacy Gitbook links.  We do need to update these, but we _also_ need to add new redirects from the _current_ URLs as well.

Can you start a new comment-block-ish section somewhere labeled "2020 site reorganization" or soemthing along those lines, and add new redirect entries there from `/introduction/` to `/thinking-in-redux/` for these pages?

Reply:
It should be better now